multilib/multilib.py

```diff
@@ -131,7 +142,8 @@ def select(self, po):
         if MultilibMethod.select(self, po):
             return True
         if po.name.startswith('kernel'):
-            for (p_name, p_flag, (p_e, p_v, p_r)) in po.provides:
+            for prov in po.provides:
+                p_name = name_from_provides(prov)
                 if p_name == 'kernel':
                     return False
         for file in po.returnFileEntries():
```

Comment:
I don't think it does. The comment on the fakepo class mentions it's used in the tests here, but as a user of multilib as a library I would not expect having to wrap the package object. Mash doesn't do any wrapping.